onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -17,10 +17,12 @@
 
 from typing import Any, Optional, Sequence
 
+from onnx import TensorProto, helper
 from onnxscript import BOOL, INT64
 from onnxscript.onnx_opset import opset18 as op
 from onnxscript.onnx_types import TensorType
 
+const_zero_int = helper.make_tensor("zero_tensor", TensorProto.INT64, [1], [0])
```

Comment:
Just a note: for anything that is a global constant, prefer UPPER_CASE. If private, prefix with `_`.

https://google.github.io/styleguide/pyguide.html#316-naming